src/manager/authentication/authManagerTypes.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: change response interface to match your backend
+export interface ISignInResponse {
+  accessToken: string
+  fullName: string
+}
+
+export interface ISignInParams {
+  email: string
+  password: string
+  remember: boolean
+}
+
+// TODO: change response interface to match your backend
+export interface ISignUpResponse {
+  accessToken: string
+  fullName: string
+}
```

Comment:
Look into network request and see what BE is providing

Reply:
Done